joins.sql

```diff
@@ -0,0 +1,62 @@
+SELECT *
+FROM users;
+
+SELECT *
+FROM posts
+WHERE user_id = 100;
+
+SELECT posts.*, users.first_name, users.last_name
+FROM posts
+INNER JOIN users on users.id = posts.user_id
+WHERE users.id = 200;
+
+SELECT posts.*, users.username
+FROM posts
+INNER JOIN users ON users.id = posts.user_id
+WHERE users.first_name = 'Norene' AND users.last_name = 'Schmitt';
+
+SELECT username
+FROM users
+WHERE created_at > '2015-01-01';
```

Comment:
This is missing a join to posts. The question is asking for all username that made a post after the given date.